examples/experiment_management/pyproject.toml

```diff
@@ -15,5 +15,8 @@ dependencies = [
     "uvicorn",
 ]
 
+[project.scripts]
+hexperiments = "hamilton_experiments.__main__:main"
```

Comment:
this is fine. If we create a fully fledged hamilton CLI then we can always migrate this to it.